lib/avo/fields/base_field.rb

```diff
@@ -116,9 +116,9 @@ def translated_plural_name(default:)
       # Secondly we'll try to find a translation key
       # We'll fallback to humanizing the id
       def name
-        return @name if custom_name?
-
-        if translation_key
+        if custom_name?
+          return Avo::ExecutionContext.new(target: @name).handle
```

Comment:
```suggestion
          Avo::ExecutionContext.new(target: @name).handle
```

We don’t need an explicit `return` since everything is already wrapped within the `if/else` condition.